[2023-01-30 13:08:48,895] INFO Terminating process due to signal SIGTERM (org.apache.kafka.common.utils.LoggingSignalHandler)
[2023-01-30 13:08:49,262] INFO [KafkaServer id=3] shutting down (kafka.server.KafkaServer)
[2023-01-30 13:08:49,280] INFO [KafkaServer id=3] Starting controlled shutdown (kafka.server.KafkaServer)
[2023-01-30 13:08:49,622] INFO Unable to read additional data from server sessionid 0x1005b8e7feb0005, likely server has closed socket, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2023-01-30 13:08:49,630] WARN Session 0x1005b8e7feb0000 for server localhost/127.0.0.1:49618, unexpected error, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
java.io.IOException: Connection reset by peer
	at java.base/sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:356)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:77)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:365)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1223)
[2023-01-30 13:08:49,782] INFO [ZooKeeperClient Kafka server] Waiting until connected. (kafka.zookeeper.ZooKeeperClient)
[2023-01-30 13:08:49,793] INFO [ZooKeeperClient Kafka server] Connected. (kafka.zookeeper.ZooKeeperClient)
[2023-01-30 13:08:50,868] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:49618. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2023-01-30 13:08:50,871] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:49618: Connection refused (org.apache.zookeeper.ClientCnxn)
[2023-01-30 13:08:50,974] INFO [ZooKeeperClient Kafka server] Waiting until connected. (kafka.zookeeper.ZooKeeperClient)
[2023-01-30 13:08:51,718] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:49618. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2023-01-30 13:08:51,720] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:49618: Connection refused (org.apache.zookeeper.ClientCnxn)
